knnvector/params/nmslib-sift-128-l2.json

```diff
@@ -0,0 +1,29 @@
+{
+    "target_index_name": "target_index",
+    "target_field_name": "target_field",
+    "target_index_body": "indices/nmslib-index.json",
+    "target_index_primary_shards": 1,
+    "target_index_dimension": 128,
+    "target_index_space_type": "l2",
+    
+    "target_index_bulk_size": 200,
+    "target_index_bulk_index_data_set_format": "hdf5",
+    "target_index_bulk_index_data_set_path": "/tmp/sift-128-euclidean.hdf5",
```

Comment:
is this path correct?

Reply:
This path should be set by users. In future we can simplify by using corpora. For now, will keep the path as user defined.

Reply:
Does this mean the user needs to download the dataset manually, put it in the desired path and then declare it here?

Reply:
Yes. I am exploring how to download default/popular dataset using corpora. This can be added incremental